README.md

```diff
@@ -75,4 +81,4 @@ contributions, in code and via his blog, this library would not be where it stan
 [2]: https://github.com/pink-tie/MoneyBundle/
 [3]: http://symfony.com/
 [4]: https://github.com/TheBigBrainsCompany/TbbcMoneyBundle
-[5]: https://packagist.org/packages/mathiasverraes/money
```

Comment:
I'm not sure. GitHub bug may be. I tried this twice. Didn't change anything on this line. It still shows this line in diff.

It's same anyway.